apps/game_backend/lib/game_backend/utils.ex

```diff
@@ -8,4 +8,6 @@ defmodule GameBackend.Utils do
 
   def get_daily_rewards_config(),
     do: Application.get_env(:game_backend, :daily_rewards_config) |> Map.get("reward_per_day")
+
+  def convert_map_keys_to_atoms(map), do: Map.new(map, fn {k, v} -> {String.to_atom(k), v} end)
```

Comment:
Never use String.to_atom().
Add functions to explicitly create the desired atom, that way we avoid a potential DoS attack.

Reply:
:panik: yeah i knew about the backend exploding due to the usage of String.to_atom() I still need to consider the user being bad and trying to destroy things, i found a workaround for this and destroy it